lib/reversetunnel/srv.go

```diff
@@ -1030,12 +1018,7 @@ func (s *server) onSiteTunnelClose(site siteCloser) error {
 			return trace.Wrap(site.Close())
 		}
 	}
-	for i := range s.localSites {
-		if s.localSites[i].domainName == site.GetName() {
-			s.localSites = append(s.localSites[:i], s.localSites[i+1:]...)
-			return trace.Wrap(site.Close())
-		}
-	}
```

Comment:
We don't need something like this here?
```go
	if s.localSite.domainName == site.GetName() {
		s.localSite = nil
		return trace.Wrap(site.Close())
	}
```

Reply:
That is my understanding as well.  The `localSite` should be a persistent component for the lifetime of the proxy.